muffnn/mlp/mlp_classifier.py

```diff
@@ -167,7 +174,11 @@ def _is_multilabel(self, y):
         Return whether the given target array corresponds to a multilabel
         problem.
         """
-        target_type = type_of_target(y)
+        temp_y=y.copy()
```

Comment:
I think this step is still necessary because 'type_of_target' can only have two unique values. Having 3 (1,0, -1) will not result in multi class indicator outputs: https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/utils/multiclass.py#L198